tests/test_40_xarray_store.py

```diff
@@ -122,6 +123,14 @@ def test_open_datasets() -> None:
     assert res[0].attrs["GRIB_centre"] == "ecmf"
 
 
+def test_cached_geo_coords() -> None:
+    ds1 = xarray_store.open_dataset(TEST_DATA_MULTIPLE_FIELDS)
+    ds2 = xarray_store.open_dataset(
+        TEST_DATA_MULTIPLE_FIELDS, backend_kwargs=dict(cache_geo_coords=True)
```

Comment:
Since the default is True, shouldn't we be testing against the case where we set it to False? I guess ideally, test all options - (default, True and False) against each other!

Reply:
That's my bad -- I originally tested with a default of `False`